### ISSUE TITLE
Truncating exercises inside history?

### ISSUE THREAD

**Issue by twizzlebizzle** [brandonp2412/Flexify#57]
Loving the app(dropped you a small sponsor) recently started working out and wanted a simple option to track my workouts.
  
As is, its very good but wanted to improve the readability of the history page.  Maybe combine all sets of the same exercise on the same day that expand with an arrow?
  
history can be a little hard to follow.
  
![Screenshot_20240616-123822](https://github.com/brandonp2412/Flexify/assets/11423542/b8d57944-3270-4491-b678-bc46d08bb061)

**Comment by brandonp2412** [comment]
Thanks mate. I dabbled with something similar on Massive but ran into performance issues (React native is slow). This project will probably be able to do this without any issue.

**Comment by twizzlebizzle** [comment]
Hi mate, thanks for that!  Good addition IMO.
